src/main/java/io/xpring/xrpl/model/XRPTransaction.java

```diff
@@ -178,7 +219,38 @@ static XRPTransaction from(Transaction transaction) {
       }
     }
 
+    // Transactions report their timestamps since the Ripple Epoch, which is 946,684,800 seconds after
+    // the unix epoch. Convert transaction's timestamp to a unix timestamp.
+    // See: https://xrpl.org/basic-data-types.html#specifying-time
+    Integer timestamp;
+    if (getTransactionResponse.hasDate()) {
+      Integer rippleTransactionDate = getTransactionResponse.getDate().getValue();
+      timestamp = rippleTransactionDate != null
+              ? rippleTransactionDate + 946684800
+              : null;
+    } else {
+      timestamp = null;
```

Comment:
pretty sure you dont need this `null` assignment.  Uninitialized variables should be `null`.